### DISCUSSION TITLE
Error 'Failed to move folder'  with Microsoft Edge

### DISCUSSION THREAD

**Discussion by barnycloud** [floccusaddon/floccus#1403] [Q&A] [answered]
When I try to sync my Nextcloud (version 25.0.5) bookmarks (version 12.1.0) in Edge to 'Weitere Favoriten', I get the following error: Failed to move folder 3716: Can't find parent bookmark for id
For 'Favoritenleiste' everything works fine.
Used floccus addon in Edge: version 4.19.1
When using Chrome in Windows 11  everything works.

**Comment by marcelklehr** [comment, marked as answer]
Please open an issue for this.
